### ISSUE TITLE
How to make TSQL support Go?

### ISSUE THREAD

**Issue by zplzpl** [antlr/grammars-v4#3533]
Does TSQL already support Go?

**Comment by kaby76** [comment]
> Does TSQL already support Go?

Yes, the Go target works. The targets it works for are listed in the desc.xml: https://github.com/antlr/grammars-v4/blob/2a7baab04d582a48465427678fe2a83df40077e6/sql/tsql/desc.xml#L4
